packages/open-next/src/build/helper.ts

```diff
@@ -356,3 +359,18 @@ export function initOutputDir(options: BuildOptions) {
   fs.mkdirSync(buildDir, { recursive: true });
   fs.cpSync(options.tempBuildDir, buildDir, { recursive: true });
 }
+
+/**
+ * @returns Whether the edge runtime is used
+ */
+export async function isEdgeRuntime(
+  overrides: DefaultOverrideOptions | undefined,
+) {
+  if (!overrides?.wrapper) {
+    return false;
+  }
+  if (typeof overrides.wrapper === "string") {
+    return overrides.wrapper.startsWith("cloudflare");
```

Comment:
This is weird, `cloudflare-node` does run in an `edge` like runtime but is meant for the `node` runtime of next.

Reply:
fixed